packages/xo-server/package.json

```diff
@@ -61,7 +61,7 @@
     "escape-string-regexp": "^1.0.5",
     "event-to-promise": "^0.8.0",
     "exec-promise": "^0.7.0",
-    "execa": "^1.0.0",
+    "execa": "^2.0.5",
```

Comment:
This new version requires Node >= 8, please upgrade `package.json` to make this explicit for `xo-server`.

Reply:
I think we should note this breaking change in the changelog.

Add a *Breaking changes* section on top and write that `xo-server` now requires Node 8.